rmc/utils/constraint.py

```diff
@@ -1708,10 +1709,13 @@ def _missing_aa_check(ht: hl.Table) -> Union[hl.Table, None]:
         return ht
 
     # Get CDS start/stops from CDS HT
-    transcript_ht = transcript_ref.ht().key_by()
+    transcript_ht = transcript_ref.ht().key_by("transcript")
```

Comment:
The table is already keyed by transcript

Reply:
you're right -- this rekey may have been intended for `ht`

Reply:
though it looks like that isn't the case below based on line 1719